internal/elasticsearch/security/role_test.go

```diff
@@ -29,6 +29,7 @@ func TestAccResourceSecurityRole(t *testing.T) {
 				Config: testAccResourceSecurityRoleCreate(roleName),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr("elasticstack_elasticsearch_security_role.test", "name", roleName),
+					resource.TestCheckResourceAttr("elasticstack_elasticsearch_security_role.test", "description", "test description"),
```

Comment:
We'll need to add a version specific test for this which is skipped for 8.14 and lower. For example https://github.com/elastic/terraform-provider-elasticstack/blob/main/internal/elasticsearch/security/role_test.go#L53-L83